### PR TITLE
add WANDB_API_KEY secret for tests

### PR DESCRIPTION
**Pull request by tomprimozic**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -37,6 +37,8 @@ on:
         required: true
       AWS_ACCOUNT_ID:
         required: true
+      WANDB_API_KEY:
```
Comment (NakulK48) [inline .github/workflows/test.yml:40]: This bit isn't required (or indeed used - it would be `${{ inputs.WANDB_API_KEY}}`)

**Comment by NakulK48** [comment]
Be sure to test this by modifying the workflow to use version `tom/add-wandb-api-key-to-tests`

File: .github/workflows/test.yml (hunk 1 above)
Comment (tomprimozic) [reply .github/workflows/test.yml:40]: hm... I copied how e.g. `AWS_ACCOUNT_ID` is used... `${{ secrets.AWS_ACCOUNT_ID }}`

shouldn't I add `WANDB_API_KEY` to `inputs` if I wanted to use `${{ inputs.WANDB_API_KEY }}`?

**Comment by tomprimozic** [comment]
this [fixed](https://github.com/CHARM-Tx/dragonligandfold/actions/runs/3995164227/jobs/6853720993) the [failure](https://github.com/CHARM-Tx/dragonligandfold/actions/runs/3994966616/jobs/6853289905)

File: .github/workflows/test.yml (hunk 1 above)
Comment (NakulK48) [reply .github/workflows/test.yml:40]: Oh whoops this is `secrets` - yup you're right

**Review by NakulK48 — APPROVED** [review]

**Pull request merged**